### PR TITLE
Add missing alias imports; Moving to var-arg transformers

### PR DESCRIPTION
**Pull request by n3wscott**
- Add missing alias imports
- Moving to var-arg transformers

Signed-off-by: Scott Nichols <snichols@vmware.com>

### CODE REVIEW

File: pkg/binding/transformer.go [hunk 1]
```diff
@@ -28,10 +28,13 @@ func (t TransformerFactories) StructuredTransformer(encoder StructuredWriter) St
 		return nil
 	}
 	res := encoder
-	if t != nil {
+	if len(t) > 0 {
```
Comment (slinkydeveloper) [inline pkg/binding/transformer.go]: you don't need this if

File: pkg/binding/transformer.go [hunk 2]
```diff
@@ -45,10 +48,14 @@ func (t TransformerFactories) BinaryTransformer(encoder BinaryWriter) BinaryWrit
 		return nil
 	}
 	res := encoder
-	if t != nil {
-		for i, _ := range t {
-			if new := t[len(t)-i-1].BinaryTransformer(res); new != nil {
-				res = new
+	if len(t) > 0 {
```
Comment (slinkydeveloper) [inline pkg/binding/transformer.go]: you don't need this if

File: pkg/binding/transformer.go [hunk 3]
```diff
@@ -59,10 +66,12 @@ func (t TransformerFactories) BinaryTransformer(encoder BinaryWriter) BinaryWrit
 
 func (t TransformerFactories) EventTransformer() EventTransformer {
 	return func(e *event.Event) error {
-		if t != nil {
-			for _, factory := range t {
-				f := factory.EventTransformer()
-
+		if len(t) > 0 {
```
Comment (slinkydeveloper) [inline pkg/binding/transformer.go]: you don't need this if

**Comment by n3wscott** [comment]
binding.ToEvent:
```
func ToEvent(ctx context.Context, message Message, transformers ...TransformerFactory) (e ce.Event, encoding Encoding, err error) 
```

**Pull request merged**